sites/www.zooniverse.org.conf

```diff
@@ -80,12 +80,12 @@ server {
         rewrite (?i)\.(jp(e)?g|gif|png|ico|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip|json)$ https://static.zooniverse.org/$host$uri;
 
         resolver 8.8.8.8;
-        if ($request_method = GET) {
+        if ($request_method ~ ^(GET|HEAD)$) {
```

Comment:
will changing to a regex affect the path hitting making it though to the upstream proxy?

Reply:
I'm not sure. Could you bring up an EC2 instance running a build of this to test it?

Reply:
Is the easiest way sitll to build my own static image and push, then ref the custom image in operations repo for a deploy?

Reply:
Yeah that'll be quickest I think

Reply:
Just noting that as of last week you can test this with docker-compose if you rebase.

Reply:
Looks like this will work based on the host path existing in s3. Checked locally and it does work what I expect.

master
```
> curl -I --compressed --header "Host: www.zooniverse.org" localhost:8080/projects/klmasters/galay-zoo-3d
HTTP/1.1 404 Not Found
Server: nginx/1.4.6 (Ubuntu)
Date: Fri, 08 Feb 2019 16:13:50 GMT
Content-Type: text/html
Connection: keep-alive
Content-Encoding: gzip
```
branch:
```
curl -I --compressed --header "Host: www.zooniverse.org" localhost:8080/projects/klmasters/galay-zoo-3d
HTTP/1.1 200 OK
Server: nginx/1.4.6 (Ubuntu)
Date: Fri, 08 Feb 2019 16:17:33 GMT
Content-Type: text/html
Content-Length: 1820
Connection: keep-alive
x-amz-id-2: VJ/py78hhqDzeGmARFK/tXJwKGxwy5+mDNLEOOwK/jKHK58xAWgift7+AKxF2S+M/07LL0JFcZc=
x-amz-request-id: 64FAE7FF2077E27A
Cache-Control: no-cache, must-revalidate
Content-Encoding: gzip
Last-Modified: Thu, 07 Feb 2019 11:47:40 GMT
x-amz-version-id: Q_Suo3U_7dgakEtl101zBXgNhi4kyqgb
ETag: "273f4cadbd02d8c33bbadc82ff2fecd7"
Access-Control-Allow-Origin: *
Access-Control-Allow-Credentials: true
Access-Control-Allow-Methods: GET
Access-Control-Allow-Headers: DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type
X-Content-Type-Options: nosniff
Content-Security-Policy: frame-ancestors 'self' zooniverse.org *.zooniverse.org webservices.crick.ac.uk CLVD0-WS-U-T-29.thecrick.org
X-Content-Security-Policy: frame-ancestors 'self' zooniverse.org *.zooniverse.org webservices.crick.ac.uk CLVD0-WS-U-T-29.thecrick.org
X-XSS-Protection: 1; mode=block
```